halo2/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "hacspec-halo2"
+version = "0.1.0"
+authors = ["Lasse Bramer Schmidt <lassebramer@gmail.com>", "Rasmus Tomtava Bjerg <rs.bjerg@gmail.com>"]
+edition = "2018"
+
+[lib]
+path = "src/halo2.rs"
+
+[dependencies]
+hacspec-lib = { path = "../../lib" }
```

Comment:
This won't work. Please use the `git` target.

Reply:
Should be resolved now:)